packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -132,6 +132,10 @@ export interface ChatbotConversationHistoryNavProps extends DrawerProps {
   isCompact?: boolean;
   /** Display title  */
   title?: string;
+  /** Icon displayed in title */
+  icon?: React.ReactNode;
+  /** Heading header level */
+  headingHeaderLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
Just to relate this and the `title`:

```suggestion
  titleHeadingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Unless we'd want to just allow spreading TitleProps here as well, since we're importing that for the Title's used in the list headers. In that case may want to rename things a bit, maybe `titleProps` becomes `listTitleProps` or something, and this becomes `navTitleProps`?

Reply:
Good call-out; thank you!